src/node/orb.yml

```diff
@@ -3,6 +3,27 @@ version: 2.1
 description: |
```

Comment:
I thought our node orb came from eugene and has jobs and other stuff.

Reply:
this is the one from eugene @eddiewebb

Reply:
unless whatever was in a separate repo had more—but i thought that got merged into the monorepo